quakemigrate/io/data.py

```diff
@@ -616,40 +602,93 @@ def get_wa_waveform(self, tr, water_level, pre_filt,
             # Get the instrument transfer function as a PAZ dictionary
             paz = response.get_paz()
 
-            # if not velocity:
-            paz.zeros.extend([0j])
+            if not velocity:
+                paz.zeros.extend([0j])
+
             paz_dict = {'poles': paz.poles,
                         'zeros': paz.zeros,
                         'gain': paz.normalization_factor,
                         'sensitivity': response.instrument_sensitivity.value}
+
             try:
                 tr.simulate(paz_remove=paz_dict,
                             pre_filt=pre_filt,
                             water_level=water_level,
                             taper=True,
                             sacsim=True,  # To replicate remove_response()
-                            pitsasim=False,  # To replicate remove_response()
-                            paz_simulate=util.wa_response())
+                            pitsasim=False)  # To replicate remove_response()
             except ValueError as e:
                 raise util.ResponseRemovalError(e, tr.id)
         else:
             # Use remove_response(), which removes the effect of _all_ response
             # stages, including the FIR stages. Considerably slower.
+            output = "VEL"
```

Comment:
```suggestion
            output = "VEL" if velocity else "DISP"
```
Could in theory be this - not a real suggested change